.github/workflows/build-and-release.yml

```
@@ -0,0 +1,65 @@
+name: Build and Release
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
```

from PerformanceAnalyst
Goのバージョン1.15はサポートが終了しているため、最新の安定バージョンに更新することをお勧めします。これにより、セキュリティとパフォーマンスの向上が期待できます。

```suggestion
go-version: [1.16, 1.17]
```